test/integration/replication.test.ts

```diff
@@ -0,0 +1,94 @@
+import { DataTypes } from '@sequelize/core';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import {
+  beforeEach2,
+  getConnectionOptionsWithoutPool,
+  getSequelizeInstance,
+  getTestDialect,
+  getTestDialectTeaser,
+} from './support';
+
+const dialect = getTestDialect();
+describe(getTestDialectTeaser('Replication'), () => {
+  if (['sqlite', 'ibmi'].includes(dialect)) {
+    return;
+  }
+
+  describe('connection objects', () => {
```

Comment:
Do we still want to test on connection strings?

Reply:
It has been moved to unit tests. Unit tests check that connection strings are properly transformed into connection objects, then this suite tests that connection objects work properly

Reply:
Ah, I probably forgot about those by the time I looked at this file 😅